AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -9,12 +9,12 @@ import SwiftUI
 
 struct SessionHeaderView: View {
     let action: () -> Void
+    @State var items: [Any]
```

Comment:
I don't like the fact that `SessionHeaderView` requires `items` to be passed in. I'd much rather see it hold some actual type instead of `Any` (if possible) and be renamed to something like `itemsForSharing` or something similarly descriptive.